src/heading/heading.tsx

```diff
@@ -1,7 +1,9 @@
 interface HeadingProps {
-    title: string;
+  children: string;
 }
 
-const Heading = ({ title }: HeadingProps) => <h1>{ title }</h1>
+const Heading = ({ children }: HeadingProps) => {
```

Comment:
Do we have any plans for this component? It's not currently adding any value.

I would suggest deleting it, or if we want to keep it around, change it back to the `<Heading title="Eventual" />` format.

Reply:
I decided to go ahead and remove this component. [20ecdde](https://github.com/bldg14/eventual/pull/15/commits/20ecdde6e4dfb97e4e1d3328c31029ab850daae4)